server/views/temporary-accommodation/bedspace-search/index.njk

```diff
@@ -91,7 +91,7 @@
 
         {{ govukTable({
           head: tableHeadings,
-          rows: tableRows
+          rows: BedspaceSearchResultUtils.resultTableRows(results.results)
```

Comment:
@bryangaledxw before I redo my commits, is this approach right? We use a module form utils instead of a presenter. We also clean up the `bedspaceSearchResult` component that we no longer need which I missed on my first pass.